dev-misc/capture-multi-weight-tabbook-mock.R

```diff
@@ -0,0 +1,89 @@
+library(httptest)
```

Comment:
This is great. Can be used for docs :+1: this process is wildly forgettable otherwise.